app/models/user.rb

```diff
@@ -775,6 +791,7 @@ def publish_notifications_state
     if self.redesigned_user_menu_enabled?
       payload[:all_unread_notifications_count] = all_unread_notifications_count
       payload[:grouped_unread_notifications] = grouped_unread_notifications
+      payload[:new_personal_messages_notifications_count] = new_personal_messages_notifications_count
```

Comment:
Random note but this is one of those spots where we can benefit from async queries.

Reply:
Sorry on second thought, this query is so fast that we will actually not benefit from async queries.